synapse/storage/databases/main/receipts.py

```diff
@@ -153,7 +153,7 @@ def f(txn):
         }
 
     async def get_linearized_receipts_for_rooms(
-        self, room_ids: List[str], to_key: int, from_key: Optional[int] = None
+        self, room_ids: Iterable[str], to_key: int, from_key: Optional[int] = None
```

Comment:
the docstring is out of sync; should be `room_ids` in the docstring (and perhaps say something other than 'List', though that's not too critical as I think 'list' is still an intuitive word)